x-pack/plugins/observability_solution/entity_manager/server/lib/entity_client.ts

```diff
@@ -78,4 +86,18 @@ export class EntityClient {
 
     return { definitions };
   }
+
+  async startTransforms(definition: EntityDefinition) {
```

Comment:
What about renaming startEntityDefinition/stopEntityDefinition to align with the naming of other methods and abstract internals

Reply:
From the view, it might be better to use wording like enable/disable

Reply:
I can just call it `start` 😅 
Might make sense since any consumer would have some EntityClient instance, so it'd read something like: `client.start()`

Reply:
To me, that sounds more like it's starting the client, even if they called it `entityManager.start` it's not really correct since it doesn't start/stop **all** of the things the entity manager is running.
I think we have to spend the character budget and call it something longer in order to be descriptive enough :)

Reply:
I updated it to @klacabane's suggestion

I left it as start/stop, but enable/disable sound fine to me too. Less so for pause/resume, since it's kinda awkward to resume something you've never started in the first place?